docker/freebsd-extras.sh

```diff
@@ -11,8 +11,16 @@ set -euo pipefail
 . freebsd-install.sh
 
 main() {
-    setup_packagesite
-    install_freebsd_package openssl sqlite3
+    apt-get update && apt-get install --assume-yes --no-install-recommends \
```

Comment:
These should be installed by default, and we need `dnsutils` for the `dig` command. Note that the `dig` output is only machine parseable without a real parser using the `+short` flag:

```bash
dig +short $url
```